src/tcpstream.c

```diff
@@ -146,9 +146,23 @@ void *run_ntttcp_sender_tcp_stream(void *ptr)
 				if (sc->domain == AF_INET) {
 					(*(struct sockaddr_in *)&local_addr).sin_family = AF_INET; /* local_addrs[i].ss_family = AF_INET; */
 					(*(struct sockaddr_in *)&local_addr).sin_port = htons(client_port);
+					if(sc->is_client_address)
+					{
+						if (inet_pton(AF_INET, sc->client_address,  &((*(struct sockaddr_in *)&local_addr).sin_addr)) <= 0) {
+							ASPRINTF(&log, "Invalid IPv4 address or Address %s not supported", sc->client_address);
+							/* Allow to go through the default interface */
```

Comment:
Add a line of log here to indicate that testing will continue with the default interface will be used